packages/openapi-to-graphql/src/preprocessor.ts

```diff
@@ -179,6 +390,13 @@ export function preprocessOas(
         const inViewer =
           securityRequirements.length > 0 && data.options.viewer !== false
 
+        /**
+         * Whether the operation should be added as a Subscription field.
+         * By default, all operations contained in callbacks should be Subscription ?
+         * Unless it is more logical to consider the callbacks parent to be the Subscription ?
+         */
+        let isSubscription = false
+
```

Comment:
Still up to discussion, should the subscription be the operation and the callbacks would be several channels to listen to ?
I went on describing each callback as a distinct subscription, so the parent operation is never one.

Reply:
I can see this as another possibility. It really depends on how people will implement their callbacks. If the callback names are unique and provide enough context by themselves, then we can create `Subscription` fields using the raw callback names. If the names themselves do not provide enough context, then we should append the path somehow. If we expect operations to contain multiple callbacks, then we can consider creating a nested structure. 

If we had more data and a better idea of conventions and best practices, it would be easier to know what is the best way to move forward.

Reply:
Even if a `CallbackObject` as uniqueName it might still contains several paths expression, [example](https://swagger.io/docs/specification/callbacks/).
Each `PathItemObject` may have it's own parameters, requestBody and responses, so if we wish to stick to OpenAPI, the simplest case would be one `Subscription` per `PathItemObject`.
But if the resolvers are able to transform the path (expression) to create severals `Channels` to subscribe to and then to resolve the payload, it would be one `Subscription` per `CallbackObject`.